packages/api/server.js

```diff
@@ -61,75 +45,54 @@ app.get('/api/repos', async (req, res) => {
   }
 
   const jsonObject = { repoNames: repoNames };
-  // TODO allow the requester to give search/filter criterion!
+
   res
     .status(200)
     .send(jsonObject)
     .end();
 });
 
-app.get('/api/auth', (req, res) => {
+app.post('/api/repo/:orgname/:reponame/test', async (req, res) => {
+  const repoId = req.params.reponame;
+  const testname = req.query.name;
   const state = v4();
-  req.session.authState = state;
-  const url = 'http://github.com/login/oauth/authorize?client_id=' + process.env.CLIENT_ID + '&state=' + req.session.authState + '&allow_signup=false';
-  res.status(302).redirect(url);
+
+  await repo.storeTicket({
+    action: 'delete-test',
+    repoId: repoId,
```

Comment:
I think we'd be smart to store a `redirect` with the ticket too, and then the folks calling the DELETE endpoint can specify the redirect they want to land on after the delete, this way we can take the user back to the page that the initiating  the deletion on ... this is useful if you're deleting multiple tests.